### ISSUE TITLE
Incompatibility with Sodium

### ISSUE THREAD

**Issue by giuliotoplad** [LudoCrypt/Vistas#37]
Panorama Screenshots doesn't work with Sodium and Iris (even without Iris still doesn't work).
Minecraft Version: **1.21**
Fabric Version: **0.15.11**
Vistas Version: **2.7.0**
Sodium Version: **0.5.9**
Iris Version: **1.7.1**

**Comment by gniftygnome** [comment]
I will look into the sodium aspect of this; they must have changed something (I know they changed all kinds of things, but something relevant).  HOWEVER, the way we take our panoramic screenshots now is by calling the vanilla method.  If the vanilla panorama capture is broken by sodium, it is up to sodium to fix it.

As for iris, that issue already exists and is pinned, even.  https://github.com/TerraformersMC/Vistas/issues/22

**Comment by gniftygnome** [comment]
Vistas 2.7.0 with Sodium 0.5.9 on default settings correctly generates screenshots in my testing.  Could you re-confirm it is broken without Iris installed and provide a complete log or at least mod version list?

**Comment by giuliotoplad** [comment]
Sorry for the delay but I don't received the notification of your response.
So yeah It works without Iris installed, but my main point is to make those screenshot with Shaders enabled because of course there're better and I used them all the time.
So It's probably something that has been modified by Iris as you said but I think maybe a method exist to avoid that problem.
I use those type of mods to make panorama screenshots for my privare server resource pack, since It's doesn't work specifically for me I used another open source project called [Panoramica](https://github.com/Keksuccino/Panoramica), but unfortunately It's abandoned so I can't use It on Minecraft 1.21 or newer versions, so I made screenshots by downgrading a backup of my server world and take it on Minecraft 1.20.4, but of course is not gonna work in long term since if some new blocks arrived into Minecraft they're not gonna be visible anymore.
